pages/content/amp-dev/documentation/guides-and-tutorials/integrate/story-interactive.md

```diff
@@ -0,0 +1,71 @@
+---
+$title: Add Interactivity support to Web Story tools
+$order: 4
+description: 'The [amp-story-interactive](https://amp.dev/documentation/components/amp-story-interactive/?format=stories) component provides various imersive elements that can be integrated into tools. Given the wide variety of options available to customize the elements, we have detailed what tools can do to have a seamless integration into the story creation workflow'
+formats:
+  - stories 
+---
+
+Interactive experiences provide a first-class support for user interactivity in Web Stories, and are designed with customization in mind. However, all the customization options can be overwhelming for users designing stories with interactive elements. We will be covering ways tools can integrate the interactive experiences into their existing workflows, and best practices to take into account.
+
+## Import scripts
+
+Start by importing the script in the head of your document:
+
+```html
+<script async custom-element="amp-story-interactive" src="https://cdn.ampproject.org/v0/amp-story-interactive-0.1.js"></script>
+```
+
+## Understanding the elements
+
+Understanding how the elements work is crucial to integrating them successfully, so we have created [a Codepen collection](https://codepen.io/collection/DEGRLE) to host demos of the components and some of their styling variants.
+
+Check out our guides on [how to integrate polls](https://amp.dev/documentation/examples/components/amp-story-interactive-poll/?format=stories) or [how to integrate quizzes with analytics](https://amp.dev/documentation/examples/components/amp-story-interactive-quiz/?format=stories) to Web Stories.
+
+## Tips on integrating into tools
```

Comment:
This section needs more context. "tools" "editor" are very broad terms. Can you restructure to:
1. Explain what the problem is
2. Explain what you did to help solving it